panels/info/cc-info-panel.c

```diff
@@ -1469,7 +1471,12 @@ updates_apply_completed (GObject *object,
   if (error != NULL)
     {
       if (!g_error_matches (error, G_IO_ERROR, G_IO_ERROR_CANCELLED))
-        g_warning ("Failed to call Apply() on EOS updater: %s", error->message);
+        {
+          CcInfoPanel *self = user_data;
+          g_warning ("Failed to call Apply() on EOS updater: %s", error->message);
+          sync_state_from_updater (self, FALSE);
```

Comment:
I’m not sure how this will indicate a local error in the UI — it seems to just query the updater state for the remote error. What am I missing?